geocity/admin_site.py

```diff
@@ -7,12 +7,29 @@
 from django.utils.http import is_safe_url
 from two_factor.admin import AdminSiteOTPRequired, AdminSiteOTPRequiredMixin
 
+from geocity.apps.permits import models
 from geocity.apps.permits.admin import PermitsAdminSite
 
 
 # https://github.com/Bouke/django-two-factor-auth/issues/219#issuecomment-494382380
 # Remove when https://github.com/Bouke/django-two-factor-auth/pull/370 is merged
 class AdminSiteOTPRequiredMixinRedirSetup(AdminSiteOTPRequired, PermitsAdminSite):
+    def has_permission(self, request):
+        permitdepartement_qs = models.PermitDepartment.objects.filter(
```

Comment:
Add comment to explain when user is using DUO vs Django MFA

Reply:
Done